.github/CODEOWNERS

```diff
@@ -49,6 +48,9 @@
 
 # ######## Core Libraries ########
 
+# Repository configuration
+/*.*                                                               @JoshLove-msft @christothes @annelo-msft @KrzysztofCwalina @jsquire @pallavit
```

Comment:
The only thing that worries me here is that this will also catch the root directories that start with `.` - including `.github` which we probably don't care about.